DOCSP-44237 flags Option deprecation on v2.21 (#201)
mongodb/docs-php-library: source/whats-new.txt

  replaced by these new methods in a future driver release, so consider
  changing the usages in your application.

> - The ``flags`` field, used for the deprecated MMAPv1 storage engine, is now 
>   deprecated and will be removed in {+library-short+} v2.0.

Reviewer: ```suggestion
- Deprecates the ``flags`` field, used for the deprecated MMAPv1 storage engine. This option will be 
  removed in {+library-short+} v2.0.
```
